service_unavailable_page/web/public/internal/index.html

```diff
@@ -1,167 +0,0 @@
-<!doctype html>
-<html lang="en" class="govuk-template">
-  <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
-    <title>
-      This service is temporarily unavailable - Apply for Qualified Teacher
-      Status - GOV.UK
-    </title>
-    <meta name="robots" content="noindex, nofollow" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1, viewport-fit=cover"
-    />
-    <meta name="theme-color" content="#0b0c0c" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <link
```

Comment:
Why does this page have so many images?

Reply:
The service unavailable page was designed to be deployed without any dependency on our service, as the service would be down, so it needed its own copy of the GOV.UK Design system.